tests/integration/pilot/common/routing.go

```diff
@@ -452,6 +452,42 @@ spec:
 	return []TrafficTestCase{tc}
 }
 
+// useClientProtocolCases contains tests use_client_protocol from DestinationRule
+func useClientProtocolCases(apps *EchoDeployments) []TrafficTestCase {
+	var cases []TrafficTestCase
+	client := apps.PodA
+	destination := apps.PodC[0]
+	cases = append(cases,
```

Comment:
does it make sense to have a gRPC test case?

Reply:
We actually need to test both h1 and h2. So it should not be gRPC test